public/content/en/gems/uniform-function-call-syntax-ufcs.md

```diff
@@ -16,14 +22,28 @@ It is possible to write
 
     a.bar().foo()
 
+Moreover in D it is not necessary to use parenthesis for functions
+without arguments, which means that _any_ function can be used
+like a property:
+
+    import std.uni : toLower;
+    "D rocks".toLower; // "d rocks"
+
+### Importance of UFCS
```

Comment:
Hmm somehow I think headings don't fit in this section. For me it reads more like a flowing text which doesn't need more structured sections. What do you think? I would vote for not adding new sections but have the other parts added/changed.